ui/src/pages/occurrence-details/reject-id/contants.ts

```diff
@@ -0,0 +1,4 @@
+export const REJECT_OPTIONS = [
```

Comment:
Hardcoding this for now @mihow, happy to fetch from backend if we have an endpoint! :) Let me know if we have more reject options, these are the ones I could find atm.